src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -72,7 +72,7 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
             boolean sessionIsOpen, int qnIdx, int responseIdx, String courseId, int totalNumRecipients) {
         return Templates.populateTemplate(
                 FormTemplates.TEXT_SUBMISSION_FORM,
-                Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                Slots.IS_SESSION_CLOSED, sessionIsOpen ? "false" : "true",
```

Comment:
@whipermr5 Agree with your suggestion about using the `Boolean.toString()` method. Didn't strike me at all! Thanks! 😅 

However, I don't fully agree with changing to `IS_SESSION_OPEN`. Yes, your argument about making the passing around of variables more readable so that people only have to think about the negation once makes sense. 

However, the fact that we are renaming it to `IS_SESSION_CLOSED` also means, that when readers have taken into account the negation that happens here, they only have to think about it that one time, because after that the variable name makes it self-explanatory. Keep in mind that we had to add this slot in, meaning that there is no precedent for passing `IS_SESSION_OPEN` that will make readers wonder about the departure from convention. 

Here, we are setting the precedent. And this was why I thought it would be better to pass down `IS_SESSION_CLOSED` instead. Because in the end, the truth value of that statement, "is the session closed?" is what we care about in the final output (in this case, if true, disable div). 

However, I do, of course, defer to your better judgment on this. Just thought I should make one final case for my suggestion. :)